include/stl2/view/istream.hpp

```diff
@@ -19,21 +19,24 @@
 #include <stl2/detail/raw_ptr.hpp>
 #include <stl2/detail/semiregular_box.hpp>
 #include <stl2/detail/iostream/concepts.hpp>
+#include <stl2/detail/iterator/default_sentinel.hpp>
 #include <stl2/view/view_interface.hpp>
 
 STL2_OPEN_NAMESPACE {
 	namespace ext {
-		template<Semiregular Val>
+		template<Movable Val>
```

Comment:
This is somewhat problematic in that `istream_view<T>` is a `Range` but not a `View` if `T` doesn't model `Copyable`. Note that range-v3 implements a relaxation of `View` which allows move-only single-pass views and requires `Copyable` only for forward views. We need to propose either that semantic or an even weaker relaxation that doesn't require copyability for *any* views, and we need to get that done in time for C++20. (I could care less if `istream_view` supports move-only types, but enabling coroutine generators to participate in view composition is crucial.)

Reply:
Clarifying: I'll merge this as-is, but we need the relaxation proposal in hand rather than proposing an `XXX_view` type for C++20 that isn't always a view.

Reply:
> We need to propose either that semantic or an even weaker relaxation that doesn't require copyability for any views, and we need to get that done in time for C++20.

I'm more comfortable looking into this relaxation, rather than going with a blanket weakening. Does this need to be done by Kona?

Reply:
> Does this need to be done by Kona?

It's a change to an existing feature in the WD, rather than a new feature proposal, so I think we could do it whenever.